api/src/main/java/org/openmrs/validator/ConditionValidator.java

```diff
@@ -44,5 +45,12 @@ public void validate(Object object, Errors errors) {
 		if(!(object instanceof Condition)){
 			throw  new IllegalArgumentException("The object parameter should be of type " + Condition.class);
 		}
+		Condition condition = (Condition) object;
+		if(condition.getCondition() == null){
+			errors.rejectValue("condition", "Condition.conditionShouldNotBeNull", "The condition parameter is required");
```

Comment:
Can we remove "parameter" from the above error message?

Reply:
Okay. Done